src/service.ts

```diff
@@ -0,0 +1,243 @@
+// tslint:disable:no-string-literal
+
+import path = require("path");
+import fs = require("fs");
+import events = require("events");
+import http = require("request");
+import q = require("q");
+import logger from "./logger";
+import pactUtil, {DEFAULT_ARG, SpawnArguments} from "./pact-util";
```

Comment:
`DEFAULT_ARG` not being used.

Reply:
I need to fix my linter, clearly. This should be a no-brainer for automation.

Reply:
or, you can use a good IDE ;)

Reply:
I've added to `tslint.json` actually, it will fail a compile if we have dangling variables that aren't being used.